bin/install-core

```diff
@@ -120,13 +124,38 @@ mkdir -p "$DOWNLOAD_DIR"
 
 # TODO: uncomment the downloads when done testing other things
 # Download Bitcoin core checksums and signatures in background
-# TODO: these are messy make them legible.
-( cd "$DOWNLOAD_DIR"; #wget --tries=1 --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/en/download/ ||
-wget --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin https://bitcoincore.org/en/download/; ) & sig_dl=$!
+(
+    cd "$DOWNLOAD_DIR"
+    for domain in ${BITCOIN_CORE_SOURCE_DOMAINS[*]}; do
+        wget --tries=1 --no-host-directories --cut-dirs=1 --recursive \
+            --level=1 --accept=SHA256SUMS* --include-directories=bin \
+            "https://${domain}/en/download/" && break
+    done
+
+    if [ $? -ne 0 ]; then
```

Comment:
I see my script doesn't wait for `$sig_dl`, it attempts to foreground the process otherwise continues if it's done.
So you'd check the exit status of the PID variable `$core_download` or `$sig_download` right after that line if you think that is a more robust check than what I did by checking if the download folder or file existed.

Reply:
Resolved with https://github.com/BenWestgate/Bails/pull/94/commits/c2a241db58bbcd5d032271e36db659aba4df7307.